packages/core/src/types.ts

```diff
@@ -1349,3 +1350,29 @@ export interface Behavior<TEvent extends EventObject, TEmitted = any> {
   initialState: TEmitted;
   start?: (actorCtx: ActorContext<TEvent, TEmitted>) => TEmitted;
 }
+
+export type EventsFrom<T> = EventObject &
```

Comment:
Same suggestion as here: https://github.com/statelyai/xstate/pull/2391#discussion_r666537803 . I would drop the `EventObject &` bit.

Reply:
6a197fcacd22876770c350ead150ec8e1d9d13cc